MAINTAINERS.md

```diff
@@ -84,3 +84,177 @@ Submit a PR with the changes and request a core team review.
+4. Wait for the new version build to complete and then edit the main project settings.
+5. From the **Default branch** drop-down, select the new version that should correspond to "latest" and save your changes.
+
+To update the Ansible core project, do the following:
+
+1. Ensure you have maintainer access to the project.
+2. Navigate to the [ansible-core dashboard](https://app.readthedocs.org/projects/ansible-core/).
+3. Select the **Add version** button and select the new version to activate, which should be a new `stable-*` branch.
+
+### Hiding versions on Read the Docs
+
+Hiding older versions removes them from the fly-out menu on Read the Docs as well as search results.
+Older versions of the documentation that are not supported or EOL should be hidden.
+It is also possible to hide doc builds before they are released to evaluate the content.
+At release day, all that is needed is to toggle the switch so the build is no longer hidden and, in the case of the package docs, update the default branch for the project.
+
+To hide versions, do the following:
+
+1. Open the settings for the version you want to remove.
+2. Select the **Hidden** option.
+
+See [How to hide a version and keep its documentation online](https://docs.readthedocs.io/en/stable/guides/hiding-a-version.html) for more information.
+
+## Building and deploying Ansible community documentation
+
+Content available at https://ansible.readthedocs.io/projects/ansible/latest/ is built and deployed from the `ansible-documentation` repository using a GitHub workflow.
+This section explains how maintainers can build and deploy the Ansible package docs using that workflow.
+
+> Details about what each job in the workflow does is beyond the scope of this maintainer guide.
+> Refer to the comments in the workflow itself for more information.
+
+### Building community documentation
+
+The Ansible package docs build workflow allows you to run a Sphinx build with the `make webdocs` target on GitHub hosted runners.
+The workflow can build from any account or org as well as from the `ansible-documentation` repository.
+For instance, if you wanted to build the content of a branch in someone's fork, you can provide those details as inputs when running the workflow.
+
+To build the docs, do the following:
+
+1. Navigate to the [Ansible package docs build](https://github.com/ansible/ansible-documentation/actions/workflows/build-package-docs.yaml) workflow.
+2. Select the **Run workflow** drop-down to open the dialog.
+3. Provide the necessary inputs and then select **Run workflow**.
+
+> If the workflow fails for any reason, the `docs-bot` posts a message in the `#docs:ansible.com` channel on Matrix.
+
+When the workflow run is complete, you can find the output available as an artifact on the **Summary** page.
+The artifact is named **package-docs-build** and contains a tarball with the generated HTML and other build assets.
+
```

Comment:
Should we add something here about open the tarball locally and verify before approving the publish.

Reply:
Sure, I'll update with that.

Reply:
I put this detail under the **Reviewing deployments** section as an extra step.